libs/generic-view/ui/src/lib/predefined/backup/backup-create.tsx

```diff
@@ -41,20 +48,39 @@ const BackupCreateForm: FunctionComponent<Config> = ({
 
   const createBackup: ButtonAction = {
     type: "custom",
-    callback: () => setStep(Step.Password),
+    callback: () => {
+      dispatch(cleanBackupProcess())
+      setStep(Step.Password)
+    },
   }
 
   const skipPassword: ButtonAction = {
     type: "custom",
     callback: () => {
+      handleSubmit((data) => {
+        console.log(data)
+        dispatch(
+          createBackupAction({
+            features: ["CONTACTS_LIST", "MESSAGES", "CALL_LOG"],
```

Comment:
I'd use the `features` from props and comment out the unused ones from `libs/generic-view/views/src/lib/mc-overview/mc-overview.ts`